File: PYTHON/Human.py
class Human:
    # constructor,Inisialisasi atribut NIK, nama, dan jenis kelamin
    def __init__(self, NIK, nama, jenis_kelamin):
        self._NIK = int(NIK)
        self._nama = nama
        self._jenis_kelamin = jenis_kelamin

    # setter dan getter
    def getNIK(self):
        return self._NIK

    def setNIK(self, NIK):
        self._NIK = NIK

    def getNama(self):
        return self._nama

    def setNama(self, nama):
        self._nama = nama

    def getJenisKelamin(self):
        return self._jenis_kelamin

    def setJenisKelamin(self, jenis_kelamin):
        self._jenis_kelamin = jenis_kelamin

File: C++/Human.cpp
#include <iostream>
#include <string>
using namespace std;
#include "Human.h"

// constructor,Inisialisasi atribut NIK, nama, dan jenis kelamin
Human::Human(int NIK, string nama, string jenis_kelamin) {
    _NIK = NIK;
    _nama = nama;
    _jenis_kelamin = jenis_kelamin;
}

// getter dan setter
int Human::getNIK() {
    return _NIK;
}

void Human::setNIK(int NIK) {
    _NIK = NIK;
}

string Human::getNama() {
    return _nama;
}

void Human::setNama(string nama) {
    _nama = nama;
}

string Human::getJenisKelamin() {
    return _jenis_kelamin;
}

void Human::setJenisKelamin(string jenis_kelamin) {
    _jenis_kelamin = jenis_kelamin;
}
